page.codeberg.outfly.outfly.yml

```
@@ -0,0 +1,38 @@
+app-id: page.codeberg.outfly.outfly
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions: ["org.freedesktop.Sdk.Extension.rust-stable"]
```

```suggestion
sdk-extensions:
  - org.freedesktop.Sdk.Extension.rust-stable
```

It's not because yaml poor design allow this inconsistent syntax that it should be used.